backend/controllers/userAuthController.js

```diff
@@ -97,10 +97,18 @@ const login = async (req, res, next) => {
 
   try {
     const user = await User.findOne({ where: { email } });
+    if (!user) {
+      return res
+        .status(400)
+        .json({ error: "Incorrect email or password", verified: true });
```

Comment:
Because on frontend it expects verified variables on every request, in this case, if user's credentials wrong or anything wrong happens, I dont want users to see my  "resend confirmation email" section, the sections depends on it,

Reply:
@Vali-R 
can you change the object property name to more specific one instead of `verified` which is broad. something like `resend_flag: true|false` :thinking: